### PR TITLE
Added integration of plotter with gateway

### LINKED ISSUE #15: Integrate gateway with plotter

### PR DESCRIPTION
**Pull request by antoniomf97**
Added integration of plotter with gateway. Models are still very simple.
This PR closes #15.

### CODE REVIEW

File: services/gateway/server.py [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+    
+    response = graph_builder(file, configs)
 
+    image = Image.open(io.BytesIO(response))
+    image.save('services\\plotter\\tmp\\test.png')
 
-@app.post("/")
-async def plot_request(upload_file: UploadFile = File(...)):
-    return {"file": "File received."}
+    return FileResponse('services\\plotter\\tmp\\test.png')
```
Comment (easyg24) [inline services/gateway/server.py]: Later we must define the inputs and outputs of the HTTP requests.

File: services/gateway/server.py [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
-app = FastAPI(title="api")
+@app.post("/", responses = {200:{"content": {"image/png": {}}}})
+async def plot_request(
+    file: Optional[UploadFile] = File(None), 
+    configs: Optional[Json[Configurations]] = Form(None)):
+    
+    response = graph_builder(file, configs)
 
+    image = Image.open(io.BytesIO(response))
+    image.save('services\\plotter\\tmp\\test.png')
```
Comment (easyg24) [inline services/gateway/server.py]: These paths shouldn't be explicit.

File: services/plotter/plotter.py [hunk 3]
```diff
@@ -26,9 +26,15 @@ def plot_builder(data):
     plt.scatter(data.index, data.y)
 
 
-def graph_builder(data, configs):
+def graph_builder(file, configs):
     fig = plt.figure()
 
+    if not configs:
```
Comment (easyg24) [inline services/plotter/plotter.py:32]: Should the verification be on this step?

**Review by easyg24 — APPROVED** [review]
Approved with some questions to think about.

File: services/gateway/server.py (hunk 2 above)
Comment (antoniomf97) [reply services/gateway/server.py]: Later we will hide them.

File: services/plotter/plotter.py (hunk 3 above)
Comment (antoniomf97) [reply services/plotter/plotter.py:32]: I would say all services should have validation first.

**Pull request merged**